data/data/baremetal/masters/main.tf

```diff
@@ -1,3 +1,19 @@
+provider "libvirt" {
+  uri = var.libvirt_uri
+}
```

Comment:
The libvirt provider is only used to create the bootstrap VM, so do we need this here in the masters file?

Reply:
We don't need this provider in the baremetal section.

Reply:
I have removed this.